CHANGES-guidelines.md

```diff
@@ -0,0 +1,81 @@
+Major releases (e.g. 1.13.0) are treated different from patch releases (e.g. 1.13.3). We would like to focus on building up complete and correct changelog content especially for patch releases. 
+
+## Patch releases
+
+**Any user-facing change** between two patch releases must be reflected in `CHANGES.md`, in one of three sections:
+
+* _Security updates_
+* _Notable changes_
+* _Fixed and improved_
+
+### How to choose the section 
+
+* If the change fixes a security issue, and for this reason it is strongly recommended to stop using earlier versions of this release, then the change must go in the _Security updates_ section.
+* If the change modifies the installation procedure or adds a new DC/OS install-time configuration option then the change must go in the _Notable changes_ section ([example](https://github.com/dcos/dcos/blame/ba77808952a9db03cd75db0631ca42921390ca06/CHANGES.md#L18)).
+* If the change updates a component with externally maintained release notes then the change must go in the _Notable changes_ section. In particular, when updating Mesos, Marathon, or the DC/OS UI please add a line of the format `Updated to [Mesos 1.5.1-dev](public-link-to-changelog-or-release-notes)` or update the existing line. This is also the method to choose when updating third-party components such as ZooKeeper, linking to external release notes.
+* If the change provides a new capability to DC/OS then the change must go in the _Notable changes_ section.
+* Other externally visible changes go in the _Fixed and improved_ section.
+* Changes that are not externally visible do not need to go in the `CHANGES.md` file. Examples include changes that only modify repository configuration and tests, or code changes that do not change external behavior.
+
+### Format
+
+Individual entries must be separated by two newline characters, so that the Markdown source shows an empty line between entries:
+```
+* Entry with no newline characters. (DCOS_OSS_JIRA)
+<empty-line>
+* Another entry with no newline characters. (DCOS_OSS_JIRA_2)
+```
+
+Add new entries to the top of the existing content within a section. Each entry should be a proper English sentence, reflecting the change in easily comprehensible terms using the past tense. A JIRA ticket ID, if applicable, should be provided side-by-side with the entry as shown above.
+
+If the entry describes a bug fix in response to a customer operations (COPS) ticket then please also add the corresponding COPS ticket ID.
+
+### What is (not) a user-facing change?
```

Comment:
I've included a summary in the "How to choose..." section. Do we still need this section?

Reply:
no, let's remove the section